Assets/Source/Constants/GameObjects.cs

```diff
@@ -5,13 +5,18 @@
     /// </summary>
     public static class GameObjects
     {
+        // todo: Break these out into categories because this class is already getting cluttered
```

Comment:
nested static classes?

Reply:
yeah thats what the ticket says to do possibly.  We might just wanna have separate classes though because I can definitely see us having like a billion different game objects